src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -390,11 +390,21 @@ public MessageFormat(String pattern) {
      * Patterns and their interpretation are specified in the
      * <a href="#patterns">class description</a>.
      *
+     * @implSpec The default implementation throws a
+     * {@code NullPointerException} if {@code locale} is {@code null}
+     * either during the creation of the {@code MessageFormat} object or later
+     * when {@code format()} is called by the constructed {@code MessageFormat}
+     * instance and the implementation utilizes a subformat that requires
+     * localization.
+     *
      * @param pattern the pattern for this message format
      * @param locale the locale for this message format
      * @throws    IllegalArgumentException if the pattern is invalid
      * @throws    NullPointerException if {@code pattern} is
      *            {@code null}
+     * @throws    NullPointerException This method throws a
+     *            {@code NullPointerException} if {@code locale} is {@code null}
+     *            and a localized subformat is used.
```

Comment:
NPE throws tags should be combined. Applies to other locations too.

Reply:
Thanks for the follow up review, I have fixed this and the 'localized' wording. I have also updated the CSR to reflect the most recent changes.